client/my-sites/site-settings/main.jsx

```diff
@@ -1,127 +1,202 @@
+		};
+	}
+
+	componentWillMount() {
 		debug( 'Mounting SiteSettings React component.' );
-		this.props.sites.on( 'change', this._updateSite );
-	},
+		this.props.sites.on( 'change', this.updateSite );
+	}
 
-	componentWillUnmount: function() {
-		this.props.sites.off( 'change', this._updateSite );
-	},
+	componentWillUnmount() {
+		this.props.sites.off( 'change', this.updateSite );
+	}
 
-	getImportPath: function() {
-		var site = this.state.site,
-			path = '/settings/import';
+	getImportPath() {
+		var { site } = this.state;
+		let path = '/settings/import';
 
 		if ( site.jetpack ) {
-			return site.options.admin_url + 'import.php';
+			return `${ site.options.admin_url }import.php`;
 		}
 
 		return [ path, site.slug ].join( '/' );
-	},
+	}
 
-	getExportPath: function() {
-		var site = this.state.site;
-		return site.jetpack ? site.options.admin_url + 'export.php' : '/settings/export/' + site.slug;
-	},
+	getExportPath() {
+		var { site } = this.state;
+		return site.jetpack
+			? `${ site.options.admin_url }export.php`
+			: `/settings/export/${ site.slug }`;
+	}
 
-	getStrings: function() {
+	getStrings() {
 		return {
-			general: this.translate( 'General', { context: 'settings screen' } ),
-			writing: this.translate( 'Writing', { context: 'settings screen' } ),
-			discussion: this.translate( 'Discussion', { context: 'settings screen' } ),
-			analytics: this.translate( 'Analytics', { context: 'settings screen' } ),
-			security: this.translate( 'Security', { context: 'settings screen' } ),
-			'import': this.translate( 'Import', { context: 'settings screen' } ),
-			'export': this.translate( 'Export', { context: 'settings screen' } ),
+			general: i18n.translate( 'General', { context: 'settings screen' } ),
+			writing: i18n.translate( 'Writing', { context: 'settings screen' } ),
+			discussion: i18n.translate( 'Discussion', { context: 'settings screen' } ),
+			analytics: i18n.translate( 'Analytics', { context: 'settings screen' } ),
+			security: i18n.translate( 'Security', { context: 'settings screen' } ),
+			import: i18n.translate( 'Import', { context: 'settings screen' } ),
+			export: i18n.translate( 'Export', { context: 'settings screen' } ),
 		};
-	},
+	}
+
+	getSections() {
+		const { site } = this.state;
+		const { purchases, context } = this.props;
 
-	getSections: function() {
-		var site = this.state.site;
 		return {
 			general: (
 				<GeneralSettings
 					site={ site }
-					purchases={ this.props.purchases } />
+					purchases={ purchases } />
 			),
 			writing: <WritingSettings site={ site } />,
 			discussion: <DiscussionSettings site={ site } />,
 			security: <SiteSecurity site={ site } />,
 			analytics: <AnalyticsSettings site={ site } />,
-			'import': <ImportSettings site={ site } />,
-			'export': <ExportSettings site={ site } store={ this.props.context.store } />
+			import: <ImportSettings site={ site } />,
+			export: <ExportSettings site={ site } store={ context.store } />
```

Comment:
Same comment about import/export as reserved words. It's ugly, but I think they should be in quotes if we use them as attributes.